### DISCUSSION TITLE
how to get users name

### DISCUSSION THREAD

**Discussion by mominb** [pocketbase/pocketbase#1583] [Q&A]
hi
i cant figure out how to display the name from the user field in my pocketbase collection.
when I use {data.collection.user} it displays the user id not their name.
and when I write {data.collection.user.name} it displayed undefined

**Comment by ganigeorgiev** [comment]
I'm sorry, without context and information about your collections structure I'm not sure that I understand what `data.collection.user` is.

Could you provide a code sample what you are trying to do (pseudo-code is also ok)?

**Reply by ganigeorgiev** [reply]
_If `user` in the above example is a `relation` type field, you can check https://pocketbase.io/docs/expanding-relations/._

**Comment by mominb** [comment]
thanks alot
